### PR TITLE
LINK-362 | Front-page content

### PR DESCRIPTION
**Pull request by jorilindell**
## Description
Add placeholder to the front page

## Closes
[LINK-362](https://helsinkisolutionoffice.atlassian.net/browse/LINK-362)

## Screenshots
Add CTA button to the front page hero
Screenshot 2021-04-19 at 18 43 47

Show most important facts about LE
Screenshot 2021-04-19 at 18 44 27

Cards of the pages using LE as data source
Screenshot 2021-04-19 at 18 45 15

Logos of the partners
Screenshot 2021-04-19 at 18 46 42

## Testing
Feature can be tested at https://linkedcomponents-ui-feature-link-362-fron-e5.test.kuva.hel.ninja/fi

### CODE REVIEW

File: src/domain/landingPage/LandingPage.tsx [hunk 1]
```diff
@@ -1,12 +1,77 @@
...
+  const { t } = useTranslation();
+
+  const myHelsinkiRoute = `https://www.myhelsinki.fi/${locale}`;
+  const tapahtumatHelRoute = `https://tapahtumat.hel.fi/${locale}/home`;
+
   return (
     <PageWrapper>
       <Hero />
+      <Container>
+        <FormContainer>
```
Comment (Sartonon) [inline src/domain/landingPage/LandingPage.tsx:27]: FormContainer is little bit confusing here :)

**Review by Sartonon — COMMENTED** [review]
Looks good. Just thinking if FormSection is good component to be used in places where there is no form

File: src/domain/landingPage/LandingPage.tsx (hunk 1 above)
Comment (jorilindell) [reply src/domain/landingPage/LandingPage.tsx:27]: Its horrible name for it :D I implemented the component when it was only used in event form and didn't change the name. Maybe now is good time to rename it

Comment (jorilindell) [reply src/domain/landingPage/LandingPage.tsx]: Removed FormContainer component and added same functionalities to Container component

**Pull request merged**